src/bootman/kernel.c

```diff
@@ -801,6 +869,20 @@ bool boot_manager_remove_kernel_internal(const BootManager *manager, const Kerne
                 }
         }
 
+        /* Remove additional initrds */
+        if (!_remove_extra_initrds(kernel->source.initrd_file)) {
+                LOG_FATAL("Failed to remove extra initrds %s.*: %s",
```

Comment:
Are we sure this needs to be `LOG_FATAL` if removing a kernel source is only `LOG_ERROR` ? I'm pretty sure we'd continue execution in an "unhappy mode" with kernel source removal, this seems like a diversion in functionality.

Reply:
Yeah, that's a mistake. Should be `LOG_ERROR` like the rest.